### PR TITLE
feat(link): add href prop

### LINKED ISSUE #300: Missing Link href prop
TypeScript does not recognize the `href` prop from the `Link` component.

```
Type '{ href: string; }' is not assignable to type 'IntrinsicAttributes & { inline?: boolean; disabled?: boolean; visited?: boolean; ref?: HTMLParagraphElement | HTMLAnchorElement; }'.
  Property 'href' does not exist on type 'IntrinsicAttributes & { inline?: boolean; disabled?: boolean; visited?: boolean; ref?: HTMLParagraphElement | HTMLAnchorElement; }'. Did you mean 'ref'? ts(2322)
```

### PR DESCRIPTION
**Pull request by albertms10**
fixes #300

---

- feat(link): add href prop

### CODE REVIEW

File: src/Link/Link.svelte [hunk 1]
```diff
@@ -1,4 +1,10 @@
 <script>
+  /**
+   * Specify href value
+   * @type {string} [href]
+   */
+  export let href;
```
Comment (metonym) [inline src/Link/Link.svelte:6]: I know it's the same, but to stay consistent initialize `href` as `undefined`

File: src/Link/Link.svelte [hunk 2]
```diff
@@ -1,4 +1,10 @@
 <script>
+  /**
+   * Specify href value
```
Comment (metonym) [inline src/Link/Link.svelte:3]: Add a "the" after "Specify"

File: types/index.d.ts [hunk 3]
```diff
@@ -1884,6 +1884,11 @@ export class InlineNotification extends CarbonSvelteComponent {
 
 export class Link extends CarbonSvelteComponent {
   $$prop_def: {
+    /**
```
Comment (metonym) [inline types/index.d.ts:1887]: Did you manually add this? It should be generated as part of the build process from running `yarn prepack`.

**Review by metonym — CHANGES_REQUESTED** [review]
See comments

File: src/Link/Link.svelte (hunk 1 above)
Comment (albertms10) [reply src/Link/Link.svelte:6]: Well, I (badly?) supposed that not initializing a prop made it kind of mandatory. Nevertheless, I will change this!

**Review by metonym — APPROVED** [review]
Thanks for fixing this

**Pull request merged**